README.md

```diff
@@ -1,2 +1,34 @@
-edx-val
-=======
+edx-val (Video Abstraction Layer)
+=================================
+
+edx-val is a django app that creates and retrieves metadata for videos and subtitles. When creating video entries, they can be assigned to preset profiles such as 'high_quality' or 'mobile_only'. When requesting a video, the client does not need to know which profile to retrieve, but only the `edx_video_id` 
+ of that video. Since all the different profiles for that particular video is returned, the client can decide which profiles they want to use. 
+ 
```

Comment:
Newlines seem inconsistent -- please either make the paragraph all one line, or break it consistently.